opentreemap/exporter/models.py

```diff
@@ -58,3 +61,6 @@ def get_url_if_ready(self):
     def complete_with(self, filename, file_obj):
         self.outfile.save(filename, file_obj)
         self.status = self.COMPLETE
+
+    def fail(self):
+        self.status = self.UNCAUGHT_EXCEPTION_ERROR
```

Comment:
I don't think there is case where we'll use `fail` and *not* immediately save the `ExportJob`.  As such, consider refactoring this method like so:
```python
def record_failure(self):
    self.status = self.UNCAUGHT_EXCEPTION_ERROR
    self.save()
```
And omitting the calls to `job.save()` below.